### DISCUSSION TITLE
Many storage.StatVol and storage.DiskInfo requests

### DISCUSSION THREAD

**Discussion by Jazz9** [minio/minio#17811] [Q&A] [answered]
I am getting ~4000 storage.StatVol per second for the same bucket.

Not sure if it's a new bug or expected behavior. 

I get 33804 lines of this in a trace over 8 seconds.
2023-08-07T11:50:02.031 [STORAGE] storage.StatVol <hostname>:9011 /mnt/01NZAKL12/testclient bucketname 48.466µs

5226 x storage.DiskInfo
33804 x storage.StatVol
15213 x storage.ReadXL
5668 x storage.RenameData
166 x storage.DeleteVersions

## Expected Behavior
I would expect the volume availability and modification time to not need to be called so often.

## Regression
I'm not sure, I upgraded today and have noticed this issue. 

## Your Environment
ubuntu 22
6 nodes, 12 x HDD drives per server, 2 x 10Gbit
minio.RELEASE.2023-08-04T17-40-21Z

**Comment by harshavardhana** [comment, marked as answer]
This is normal you have perhaps setup a very high scanner speed that causes it to run in a continuous manner.

**Comment by Jazz9** [comment]
Thanks - the scanner was on slowest. I wonder what is causing it. I'll try and setup a isolated test.

**Reply by harshavardhana** [reply]
Then there are other factors also involved such as Prometheus monitoring etc that will hit all the volumes etc.
